### DISCUSSION TITLE
how using Microsoft.Expression.Drawing to Uno

### DISCUSSION THREAD

**Discussion by AbdalaMask** [unoplatform/uno#5074] [Q&A] [answered]
how using Microsoft.Expression.Drawing to Uno 

using BlockArrow and RegularPolygon and Arc

**Comment by jeromelaban** [comment, marked as answer]
There is currently no support for `Microsoft.Expression.Drawing`, and it is not supported by WinUI as well: https://github.com/microsoft/microsoft-ui-xaml/issues/2621
